cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/services/RunRecordCounter.java

```diff
@@ -71,20 +71,32 @@ public Count addRequestAndGetCount(ProgramRunId programRunId) {
       list.stream().filter(r -> isRunning(r.getController().getState().getRunStatus())).count()
       + launchingCount;
 
-    LOG.debug("Adding request with runId {}. Counter now has {} concurrent launching and {} concurrent runs.",
-              programRunId, launchingCount, runsCount);
+    LOG.debug("Counter has {} concurrent launching and {} concurrent runs.", launchingCount, runsCount);
```

Comment:
can we make such log info. Usually when problem happen , we need to look into the history logs before changing the log level to debug.